client/src/annoMatrix/normalize.ts

```diff
@@ -91,12 +102,11 @@ export function normalizeWritableCategoricalSchema(colSchema: any, col: any) {
   return colSchema;
 }
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
 export function normalizeCategorical(
   df: Dataframe,
```

Comment:
the use of DataFrame here is also referring to the schema sub-object, rather than an instance of class DataFrame (it should be the latter). Should be `any` until `util/dataframe` is typed.